app/utils/display-converter.js

```diff
@@ -138,9 +139,13 @@ module.exports = function (req, data, template) {
   convertedData.isLive = req.isLive
   convertedData.humanReadableEnvironment = convertedData.isLive ? 'Live' : 'Test'
   const currentPath = (relativeUrl && url.parse(relativeUrl).pathname.replace(/([a-z])\/$/g, '$1')) || '' // remove query params and trailing slash
+  const currentUrl = req.baseUrl && req.baseUrl ? req.baseUrl + req.path : 'unavailable'
```

Comment:
Is the `req.baseUrl &&` necessary if it's also in the ternary? i.e. would just the ternary
`req.baseUrl ? req.baseUrl + req.path : 'unavailable'`
be equivalent

Reply:
ah good spot, it's supposed to be `req.baseUrl && req.path...`